doc/source/ndl.rst

```diff
@@ -11,7 +33,7 @@ In order to update the association strengths (weights) between cues and
 outcomes we do for each event the following:
 
 We calculate the activation (prediction) :math:`a_j` for each outcome
-:math:`o_j` by using all present cues :math:`C_\text{PRESENT}`: 
+:math:`o_j` by using all present cues :math:`C_\text{PRESENT}`:
 
 .. math::
```

Comment:
In Quickstart section and more example section another version of the formula is used. We should stay consistent. 
Is it possible to import it at all three positions from a separate file?

Reply:
I opened an issue for that (#133) to resolve it later